### DISCUSSION TITLE
How to Prevent Users from Creating Accounts

### DISCUSSION THREAD

**Discussion by yottabit42** [mealie-recipes/mealie#1693] [General]
I like to run Mealie public-facing so we can always access our recipes from anywhere, but with 1.0 it seems there are a few issues compared to what I was used to in the earlier release:

1. The landing page does not show the recipes publicly (I understand this is a WIP and something like this will be added soon)
2. Any random person can create an account on my server

I'm most concerned with 2 above. How can I prevent random people from creating accounts on my server? :-)

I will close it to the outside, and add a VPN to my wife's phone if I have to, but I'm hoping there's an option in the settings I've missed somewhere...

**Comment by Kuchenpirat** [comment]
There is no deployment detail but i assume you are using a docker compose file. 
There you can change the value of the env variable "ALLOW_SIGNUP" from _true_ to _false_ to disable signup without invitation on your server. 

Hope this helps.

**Reply by yottabit42** [reply]
Awesome, thanks for the pointer. That did the trick!

**Reply by Kuchenpirat** [reply]
Yeah. Glad to help
